scanners/boostsecurityio/brakeman/module.yaml

```
@@ -12,7 +12,7 @@ steps:
   - scan:
       command:
         docker:
-          image: presidentbeef/brakeman:latest@sha256:7416e4cf46131d5f920be496485d30d55a9b9f00acec28847ae1e5f10ac837f4
+          image: presidentbeef/brakeman:v6.0.0.1@sha256:511c731712b4252d7056511e67051a498fe333b956ba89b19e1e938de363a94a
```

This appears to be the digest of the amd64 image. Use the manifest digest so the image can work on arm too:

```
docker buildx imagetools inspect "presidentbeef/brakeman:v6.0.0.1"
Name:      docker.io/presidentbeef/brakeman:v6.0.0.1
MediaType: application/vnd.docker.distribution.manifest.list.v2+json
Digest:    sha256:78add476199155ef44d77509536ab6b0819a8d2fce7e11f2da1d22aa26c5e6b4
           
Manifests: 
  Name:      docker.io/presidentbeef/brakeman:v6.0.0.1@sha256:511c731712b4252d7056511e67051a498fe333b956ba89b19e1e938de363a94a
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/amd64
             
  Name:      docker.io/presidentbeef/brakeman:v6.0.0.1@sha256:706194e0c47af7e2c9cc098aa28bc7221e58dd64cd4932809b4bd2a41f921696
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/arm64
```

```suggestion
          image: presidentbeef/brakeman:v6.0.0.1@sha256:78add476199155ef44d77509536ab6b0819a8d2fce7e11f2da1d22aa26c5e6b4
```